src/material-experimental/mdc-button/fab.scss

```diff
@@ -9,9 +9,13 @@
 .mat-mdc-fab, .mat-mdc-mini-fab {
   @include _mat-button-interactive();
   @include _mat-button-disabled();
-}
 
-.mat-mdc-fab, .mat-mdc-mini-fab {
+  // MDC adds some styles to fab and mini-fab that conflict with some of our focus indicator
+  // styles and don't actually do anything. This undoes those conflicting styles.
+  &:not(.mdc-ripple-upgraded):focus::before {
```

Comment:
@andrewseguin do you know why these styles are added?

I suspect they may be added for the sake of the MDC ripples, which we currently don't use but would like to in the future.

Reply:
I believe this is okay, since I've only seen MDC use the `before` and `after` of the ripple element, not the host element. I'd leave the comment just in case, but I think this is safe to do